src/AdvApi32.Desktop/AdvApi+SE_OBJECT_TYPE.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="SE_OBJECT_TYPE"/> nested type.
+    /// </content>
+    public partial class AdvApi32
+    {
+        /// <summary>
+        /// The SE_OBJECT_TYPE enumeration contains values that correspond to the types of Windows objects that support security.
+        /// The functions, such as <see cref="GetSecurityInfo(System.IntPtr, SE_OBJECT_TYPE, SECURITY_INFORMATION, System.IntPtr, System.IntPtr, Kernel32.ACL*, Kernel32.ACL*, Kernel32.SECURITY_DESCRIPTOR*)"/> and <see cref="SetSecurityInfo(System.IntPtr, SE_OBJECT_TYPE, SECURITY_INFORMATION, System.IntPtr, System.IntPtr, Kernel32.ACL*, Kernel32.ACL*)"/>, that set and retrieve the security information of an object,
+        /// use these values to indicate the type of object.
+        /// </summary>
+        public enum SE_OBJECT_TYPE
```

Comment:
The file name for this object should be `AdvApi32+SE_OBJECT_TYPE.cs` instead of `AdvApi+SE_OBJECT_TYPE.cs`